nix/server.nix

```diff
@@ -117,8 +118,8 @@ in
   networking = {
     hostName = "quid"; # Define your hostname.
     hostId = "e39841f0";
-    firewall.allowedTCPPorts = [ 80 443 3000 8888 42063 8000 ];
-    firewall.allowedUDPPorts = [ 80 443 3000 8888 42063 8000 ];
+    firewall.allowedTCPPorts = [ 80 443 3000 8888 42063 8000 8080 ];
```

Comment:
Is exposing this port actually necessary? nginx should be proxying from 80/443....